airflow/opt/providers/etna/etna/etls/metis.py

```diff
@@ -243,7 +244,7 @@ def _expand_project_parents(self, update: UpdateRequest, models: Dict[str, Model
                 continue
             model = models[model_name]
             self.log.info(f"Expanding project={project} for model {model_name}...")
-            if model.template.parent == "parent":
+            if model.template.parent == "project":
```

Comment:
oooh is this where my xneo1 linking went wrong?

Reply:
Yessir!